src/main/resources/swagger.api/inn-reach.yaml

```diff
@@ -5,175 +5,161 @@ info:
+      parameters:
+        - $ref: "#/components/parameters/trait_pageable_offset"
+        - $ref: "#/components/parameters/trait_pageable_limit"
+  /locations/{locationId}:
+    get:
+      responses:
+        '200':
+          description: OK
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Bad request
+                $ref: "#/components/schemas/innReachLocationDTO"
         '404':
-          description: Configuration not found
+          description: InnReach location not found
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Configuration not found
+                $ref: "#/components/schemas/error"
         '500':
           description: Internal server error
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Internal server error
-      description: Change the inn reach configuration
-      operationId: putConfiguration
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: "#/components/schemas/moduleConfiguration"
-        required: true
-  /configurations/{configId}:
-    get:
+                $ref: "#/components/schemas/error"
+      description: Get InnReach location by id
+      operationId: getLocationById
+      parameters:
+        - $ref: '#/components/parameters/locationId'
+    put:
       responses:
         '200':
-          description: ""
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/innReachLocationDTO"
+        '400':
+          description: Bad request
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/moduleConfiguration"
+                $ref: "#/components/schemas/validationErrorsDTO"
         '404':
-          description: Configuration not found
+          description: InnReach location not found
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Configuration not found
+                $ref: "#/components/schemas/error"
         '500':
           description: Internal server error
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Internal server error
-      operationId: getConfigurationById
+                $ref: "#/components/schemas/error"
+      description: Update InnReach location
+      operationId: updateLocation
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/innReachLocationDTO"
```

Comment:
The same here: do we need to mention module's name when specify schema name for payload definition here? Or we can have several kind of locations like "InnReach" and "CentralServers" for example and therefore needs to distinguish them?